### ISSUE TITLE
Scala Index version is 1.1.3 but documentation is about 1.1.0

### ISSUE THREAD

**Issue by angelcervera** [altoo-ag/akka-kryo-serialization#207]
There is a little bit of mess in the README.md, in relation to the version.
In the documentation, all references are about 1.1.0, but the last version following the label of scala index is 1.1.3
![image](https://user-images.githubusercontent.com/293444/76656251-1c5b3e00-6567-11ea-8f3a-b2819d821422.png)

If 1.1.3 is the last version, it is necessary to change:
- The url pointing to the Maven repository to https://repo1.maven.org/maven2/com/github/enalmada/akka-kryo-serialization_2.12/1.1.3/
- the sbt dependency package to "`com.github.enalmada" %% "akka-kryo-serialization" % version`
- There is no tag for version 1.1.3 in Git.

If the last version is 1.1.0, it is necessary to change the scala index label.

**Comment by danischroeter** [comment]
Hi @angelcervera 
Thx for the info. The latest release of io.altoo:akka-kryo-serialization is 1.1.0 and all the tags/code you see is up to date.

There is however @enalmada who has made a clone of the repo and released the same project (looks like he tested releasing since there are no changes).
Somehow he was allowed by sonatype to upload these to maven central :(

Now the sacadex scanner downloads these fake packages that are valid poms but somehow associates them with this project.
```
    <groupId>com.github.enalmada</groupId>
    <artifactId>akka-kryo-serialization_2.13</artifactId>
    <packaging>jar</packaging>
    <description>akka-kryo-serialization</description>
    <version>1.1.3</version>
    <name>akka-kryo-serialization</name>
    <organization>
        <name>com.github.enalmada</name>
    </organization>
    <url>https://github.com/altoo-ag/akka-kryo-serialization</url>
```

Hmm this is very annoying. This looks like a bug in the scaladex scanner.
Other than that: nobody should upload code under io.altoo to the sonatype repo.

**Comment by Enalmada** [comment]
I have a bunch of projects using Play Framework which wont start unless all dependencies use its exact version of akka which they like to change alot.  So when they release I update the modules I need and publish to my own org.   Is there something I could do to make scaladex scanner ignore my stuff or minimum changes to make it not associated with the original fork like leaving url blank or something?

**Comment by danischroeter** [comment]
@Enalmada Instead of releasing the projects with different dependencies you can remove/ignore the dependency so they are only taken from play. 

This is what we for example do to not get akka from kamon. You should do this for all projects except for the one which you want to match the akka depencency from.
```
"io.kamon" %% "kamon-akka-http" % kamonAkkaHttpV excludeAll(ExclusionRule("io.kamon", "kamon-core_2.13"), ExclusionRule("com.typesafe.akka", "akka-actor_2.13"), ExclusionRule("com.typesafe.akka", "akka-cluster_2.13"), ExclusionRule("com.typesafe.akka", "akka-remote_2.13"), ExclusionRule("com.typesafe.akka", "akka-http_2.13"))
```

**Comment by danischroeter** [comment]
@Enalmada this should be ready to go:
```
lazy val akkaKryoSerializationDeps = Seq(
  "io.altoo" %% "akka-kryo-serialization" % "1.1.0" excludeAll (ExclusionRule("com.typesafe.akka", "akka-actor_2.13")),
)
```

**Comment by danischroeter** [comment]
@Enalmada I found the reason why your artifacts are wrongly linked - you need to fix the scm tags in the poms - this is what scaladex uses to link... 

This is from your pom which should link to your own git repo:
```
<scm>
<url>
git@github.com:altoo-ag/akka-kryo-serialization.git
</url>
<connection>
scm:git:git@github.com:altoo-ag/akka-kryo-serialization.git
</connection>
</scm>
```

**Comment by danischroeter** [comment]
@nvollmar we might need to create a version > 1.1.3 to "fix" this issue

**Comment by aluveitie** [comment]
Maybe Kryo 5 is finally released a new RC has been published a couple of weeks ago

**Comment by danischroeter** [comment]
fixed by rolling out a 1.1.5 version

**Comment by Enalmada** [comment]
I tried the `excludeAll (ExclusionRule("com.typesafe.akka", "akka-actor_2.13"))` and it seems to work.  Thanks for that better way to handle this!
